editor/example/index.ts

```diff
@@ -105,25 +128,29 @@ export const examples: {
     VISUAL_ENCODING: {
         group: 'Visual Encoding',
         name: 'Visual Encoding',
-        spec: EX_SPEC_VISUAL_ENCODING,
+        spec: JSON_SPEC_VISUAL_ENCODING as GoslingSpec,
+        specJs: jsStringParser(JS_SPEC_VISUAL_ENCODING),
         image: THUMBNAILS.VISUAL_ENCODING
     },
     VISUAL_ENCODING_CIRCULAR: {
         group: 'Visual Encoding',
         name: 'Circular Visual Encoding',
-        spec: EX_SPEC_VISUAL_ENCODING_CIRCULAR,
+        spec: JSON_SPEC_VISUAL_ENCODING_CIRCULAR as GoslingSpec,
+        specJs: jsStringParser(JS_SPEC_VISUAL_ENCODING_CIRCULAR),
         image: THUMBNAILS.VISUAL_ENCODING_CIRCULAR
     },
     BAND: {
         group: 'Visual Encoding',
         name: 'Band Connection',
-        spec: EX_SPEC_BAND,
+        spec: JSON_SPEC_BAND as GoslingSpec,
+        specJs: jsStringParser(JS_SPEC_BAND),
         image: THUMBNAILS.BAND
     },
     RULE: {
         group: 'Visual Encoding',
         name: 'Rule Mark',
-        spec: EX_SPEC_RULE,
+        spec: JSON_SPEC_RULE as GoslingSpec,
```

Comment:
We can remove all `as GoslingSpec` when we define types of a variable `spec` in individual example files.

```suggestion
        spec: JSON_SPEC_RULE,
```